.github/workflows/dockerd.yml

```diff
@@ -0,0 +1,175 @@
+name: dockerd
+
+on:
+  schedule:
+    - cron: '0 10 * * *'
```

Comment:
Maybe remove cron as this can't pass without moby vendor.

Reply:
Yes agree it will failed with `20.10.13`